server/controllers/questions.js

```diff
@@ -0,0 +1,167 @@
+const axios = require('axios');
+const baseURL = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp';
+require('dotenv').config();
+
+module.exports = {
+  getAllQuestionsByProductId: (req, res) => {
+    // page&count has default value 1&5.
+    // keep this in mind in case you need to change these value.
+    const {page, count, product_id} = req.query;
+    let url = `/qa/questions?page=${page}&count=${count}&product_id=${product_id}`;
+    let config = {
+      method: 'get',
+      baseURL: baseURL,
```

Comment:
I think the airbnb linter would flag this notation. 

Please change baseURL and url keys to use object shorthand.

https://eslint.org/docs/latest/rules/object-shorthand

Reply:
Thank you Kev! :)